cobalt/precommit/run_chromium_presubmits.py

```diff
@@ -0,0 +1,147 @@
+    "CheckAuthorizedAuthor",
+    "CheckChangeWasUploaded",
+    "CheckLicense",
+    "CheckOwners",
+    "CheckOwnersFormat",
+    "CheckOwnersOnCommit",
+    "CheckOwnersOnUpload",
+]
+
+# --- Monkey-patching to remove Gerrit dependencies ---
+# pylint: disable=protected-access
+original_run_check_function = \
+  presubmit_support.PresubmitExecuter._run_check_function
+
+# pylint: enable=protected-access
+
+
+# Define the patched version of _run_check_function to disable OWNERS checks
+def filtered_run_check_function(self, function_name, context, sink,
+                                presubmit_path):
+  """A patched version that disables OWNERS checks which require Gerrit."""
+  if function_name in SKIPPED_PRESUBMIT_CHECKS:
+    # Return an empty list to mimic a successful check
+    return []
+  # Call the original function for all other checks
+  return original_run_check_function(self, function_name, context, sink,
+                                     presubmit_path)
+
+
+presubmit_support.PresubmitExecuter._run_check_function = \
+  filtered_run_check_function  # pylint: disable=protected-access
+
+
+def no_op_check_function(*_args, **_kwargs):
+  """A no-op version for check function."""
+  return []
+
+
+# Presubmit checks that are called from other checks can't be directly blocked
+# with our monekypatched solution, so we also have to monkeypatch the original
+# module. For now just check on presubmit_canned_functions by default
+for check in SKIPPED_PRESUBMIT_CHECKS:
+  if hasattr(presubmit_canned_checks, check):
+    setattr(presubmit_canned_checks, check, no_op_check_function)
+
+
+def get_file_changes_for_presubmit(file_paths):
+  """
+    Converts a list of file paths into the format expected by presubmit_support.
+    Change. For now, it assumes all files are 'Modified'. A more advanced
+    version could use git to determine the actual status (Added, Modified,
+    Deleted).
+    """
+  changes = []
+  for path in file_paths:
+    if os.path.exists(path):
+      # Assume 'M' for any existing file passed by the hook.
+      # 'A' would also work. 'D' would require more logic.
+      changes.append(("M", path))
+  return changes
+
+
+def main(argv: List[str]) -> int:
+  """
+    Main function to execute Chromium presubmit checks on files
+    staged in pre-commit.
+    """
+  if shutil.which("gclient") is None:
+    print("depot_tools doesn't seem to be installed.")
+    sys.exit(1)
+
+  if not argv:
+    print("No staged files to check.")
+    sys.exit(0)
+
+  # --- Prepare the Change object ---
+  repository_root = os.getcwd()
+  changed_files = get_file_changes_for_presubmit(argv)
+  change = presubmit_support.GitChange("Pre-commit Change",
+                                       "Files staged for commit.",
+                                       repository_root, changed_files, None,
+                                       None, "fake@user.com")
+
+  # --- Run the presubmit checks ---
+  committing = True
```

Comment:
You could just put these values directly in the function call instead of creating variables for them